src/content/tools/analyzer-plugins.md

```diff
@@ -189,6 +199,17 @@ check out the following guides:
 [Writing assists]: https://github.com/dart-lang/sdk/blob/main/pkg/analysis_server_plugin/doc/writing_assists.md
 [Testing rules]: https://github.com/dart-lang/sdk/blob/main/pkg/analysis_server_plugin/doc/testing_rules.md
 
+Check the Dart Analysis Server for many examples of rules,
+fixes, and, assists:
+
+* [Dart Analysis Server's rules][]
+* [Dart Analysis Server's fixes][]
+* [Dart Analysis Serer's assists][]
```

Comment:
The guides in the list above link to the Dart Analysis Server's rules/fixes/assists. These are very useful, so I bubbled these up here to raise awareness.

Reply:
Nice!